### PR TITLE
Datahub: use maxExtent from mapConfig as extent fallback

### PR DESCRIPTION
**Pull request by tkohr**
If the extent of a layer displayed on the map, cannot be determined, the PR:

- uses the `maxExtent` param (if present) from the config's `map` section as a fallback extent
- otherwise applies a default value in `mapcontext service`

### CODE REVIEW

File: libs/feature/map/src/lib/map-context/map-context.service.ts [hunk 1]
```diff
@@ -30,6 +31,11 @@ export const DEFAULT_BASELAYER_CONTEXT: MapContextLayerXyzModel = {
   ],
 }
 
+export const DEFAULT_VIEW: MapContextViewModel = {
+  center: fromLonLat([2.1, 46.8], 'EPSG:3857'),
```
Comment (jahow) [inline libs/feature/map/src/lib/map-context/map-context.service.ts:35]: maybe it would be a good time to not have metropolitan France as a default extent any more :) a global world view would make more sense IMO

**Review by jahow — APPROVED** [review]
Thanks, added a couple of comments, please feel free to merge when you think it's good!

File: libs/feature/map/src/lib/map-context/map-context.service.ts (hunk 1 above)
Comment (tkohr) [reply libs/feature/map/src/lib/map-context/map-context.service.ts:35]: true, I'll adapt this, only drawback will be that datasets in France for which the viewer falls back to this default, will become less "visible"

File: libs/feature/map/src/lib/map-context/map-context.service.spec.ts [hunk 2]
```diff
@@ -311,6 +312,71 @@ describe('MapContextService', () => {
         expect(baselayerUrls).toEqual(DEFAULT_BASELAYER_CONTEXT.urls)
       })
     })
+    describe('uses default fallback view (without config)', () => {
+      let view
+      const map = new Map({})
+      const mapContext = MAP_CTX_FIXTURE
+      mapContext.view.extent = null
```
[the remark this answers is not in the document]
Comment (tkohr) [reply libs/feature/map/src/lib/map-context/map-context.service.spec.ts:319]: ah, indeed this is what made the CI crash, thanks

File: libs/feature/record/src/lib/data-view-map/data-view-map.component.spec.ts [hunk 3]
```diff
@@ -609,7 +609,7 @@ describe('DataViewMapComponent', () => {
       })
       describe('when extent computation fails', () => {
         beforeEach(inject([MapUtilsService], (mapUtils) => {
-          mapUtils._observer.error('extent computation failed')
+          mapUtils._observer.next(null)
```
[the remark this answers is not in the document]
Comment (tkohr) [reply libs/feature/record/src/lib/data-view-map/data-view-map.component.spec.ts:612]: ah right, I add another test for the case that the calculation returns `null`
